### DISCUSSION TITLE
Meal Planner Sets

### DISCUSSION THREAD

**Discussion by BrandonG777** [mealie-recipes/mealie#2492] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Add the ability to save a Meal Planner (date range) as a set to easily reuse later. 

### Please provide a concise description of the feature that would resolve your issue.

This would be especially useful during holidays, to save a series of meals that are regularly prepared around the holidays.

### Please consider and list out some caveats or tradeoffs made in your design decision

Maybe add a "Save" button next to the settings button? And then maybe another icon (next to the random and + buttons) to re-add the saved set?

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [X] I'm willing to sponsor/pay a developer to do this work

**Comment by Acidherr** [comment]
I like this idea, I feel it would be useful, get three meals that you can quick together in a meal set and just whip it out in days where you have lots to do.
